test/battle/move_effect/solar_beam.c

```diff
@@ -0,0 +1,183 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_SOLAR_BEAM].effect == EFFECT_SOLAR_BEAM);
+}
+
+SINGLE_BATTLE_TEST("Solar Beam can be used instantly in Sunlight")
```

Comment:
Combine the separation of Solar Beam and Blade into the same tests using `PARAMETRIZE`

Reply:
Haven't addressed this yet.